test/unit/core/entities/ds_data_type_test.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+
+void main() {
+  group('DsDataType', () {
```

Comment:
add test to verify that class has all necessary names with proper values:
  bool('bool', 1),
  integer('int', 2),
  uInt('uint', 2),
  dInt('dint', 4),
  word('word', 2),
  lInt('lint', 8),
  real('real', 4),
  time('time', 4),
  dateAndTime('date_and_time', 8);

Reply:
Done